char/liberate/NETLIST/gf180mcu_osu_sc_12T_addf_1.spice

```diff
@@ -0,0 +1,42 @@
+* HSPICE file created from gf180mcu_osu_sc_12T_addf_1.ext - technology: gf180mcuC
+
+.inc "/import/yukari1/lrburle/OSU_180/char/techfiles/design.hspice"
```

Comment:
`yukari1/lrburle` sounds like hostname/username? are we sure we want to keep this as it in the repo?

Reply:
fixed